CONTRIBUTE.md

```diff
@@ -153,3 +153,18 @@ let sol = decomp
      .solve_triangular(UPLO::Lower, Diag::NonUnit, &Array::eye(n_features))?
      .without_lapack();
 ```
+
+## Benchmarking
+
+It is important to the project that we have benchmarks in place to evaluate the benefit of performance related changes. To make that process easier we provide some guidelines for writing benchmarks.
+
+1. Test for a variety of sample sizes for most algorithms [1_000, 10_000, 20_000] will be sufficient
+2. Test of variety feature dimensions
+    - Spatial algorithms: [3, 5, 8]
+    - Dimentionality reduction algorithms: [10, 20, 50]
+    - Others: [5, 10]
+3. Use Criterion
+4. Test various alg implementations
+5. Set a random seed for algorithm if applicable
```

Comment:
"For algorithms that require an RNG or random seed as input, use a constant seed for reproducibility,